packages/react-data-grid/style/rdg-header.less

```diff
@@ -10,12 +10,14 @@
 }
 
 .react-grid-HeaderRow {
-  position: relative;
-  overflow: hidden;
+  position: sticky;
```

Comment:
So you plan to stick the header rows like normal rows? How will the canvas vertical scroll bar look like?

Reply:
The scrollbar will cover the header rows, I think that's fine.

Reply:
hmm... seriously, I had never seen a grid columns are covered by it's rows canvas vertical scroll bar.

https://github.com/FancyGrid/awesome-grid

Reply:
Why spend more effort on such a benign issue?

Reply:
For IE 11 and IE Edge, vertical scrolling will make the headers and summary rows jumping around after setting the `transform: translateY()`.

Just FYI, here is how it looks like for IE11 and IE Edge.
IE 11 (header columns are scrolled to top, with the transform calculated later, it will be scrolled to top and jump back)
![IE 11 vertical scrolling](https://user-images.githubusercontent.com/25043398/66423731-e4593900-e9d1-11e9-8030-cd8d82ccb5a1.gif)

IE Edge (header columns are gone and reappear)
![IE Edge vertical scrolling](https://user-images.githubusercontent.com/25043398/66423801-02bf3480-e9d2-11e9-84d7-a6e8d33e3b7f.gif)

Reply:
This pr isn't ready for review.